app/app.ts

```diff
@@ -0,0 +1,10 @@
+import { Generator } from './utils/index';
+
+try {
+  const idGenerator = new Generator(1, 10, 1);
+  const id = idGenerator.getId();
+
+  console.log(`ID: ${id}`);
+} catch (error) {
+  console.log(error);
+}
```

Comment:
Not sure what this file is for, I guess for manual testing. Would you leave it here if you were to publish this to npm? 😉